loadbalancer/zk/client.go

```diff
@@ -0,0 +1,82 @@
+package zk
+
+import (
+	"github.com/go-kit/kit/log"
+	"github.com/samuel/go-zookeeper/zk"
+	"strings"
+	"time"
+)
+
+const ()
+
+// Client is a wrapper arround the zookeeper client.
+type Client interface {
+	// GetEntries queries the path in zookeeper, places a watch on it
+	// and retrieves data from its current child nodes
+	GetEntries(path string) ([]string, <-chan zk.Event, error)
+	// createParentNodes tries to create the path
+	// in case it does not exist yet on Zookeeper
+	createParentNodes(path string) error
+}
```

Comment:
I've never seen an exported interface with both exported and nonexported methods. Do you have some precedent for that? Unless there's something I'm overlooking, I think it must be a mistake. If the interface exists because you intend other packages to implement Client, the nonexported method prevents that. And if you intend the nonexported method to be for package zk internal usage only, then there's no reason to export Client...